frontend/src/pages/projects/screens/detail/notebooks/NotebookTableRow.tsx

```diff
@@ -51,7 +43,14 @@ const NotebookTableRow: React.FC<NotebookTableRowProps> = ({
   const { currentProject } = React.useContext(ProjectDetailsContext);
   const navigate = useNavigate();
   const [isExpanded, setExpanded] = React.useState(false);
-  const { size: notebookSize, error: sizeError } = useNotebookDeploymentSize(obj.notebook);
```

Comment:
So you didn't address the `error` source here... you replaced the only code that used it -- and technically there are still avenues to address an error here. Please look at the `useNotebookDeploymentSize` and determine if you need to make changes to satisfy this.